StorageTests/TestSQLiteHistory.swift

```diff
@@ -1292,7 +1342,8 @@ class TestSQLiteHistory: XCTestCase {
 
             return history.getTopSitesWithLimit(20) >>== { topSites in
                 XCTAssertEqual(topSites.count, 20)
-                XCTAssertEqual(topSites[0]!.guid, "abc\(0)def")
+                XCTAssertEqual(topSites[0]!.guid, "abc\(5)def")
+                XCTAssertEqual(topSites[1]!.guid, "abc\(0)def")
```

Comment:
Yes: the `ORDER BY` clause doesn't result in a total order. We only order by `is_bookmarked` and `frecencies`, so anything after that is arbitrary.

If you want to make this consistent, you could add a descending grouping on domain ID or history ID, which will break ties for most recent domain or item.